### DISCUSSION TITLE
Add option to play from file with the web dashboard

### DISCUSSION THREAD

**Discussion by Jigsaw88** [aldostools/webMAN-MOD#968] [Polls]
Dankö

**Comment by aldostools** [comment]
What do you mean exactly? I think this feature is already implemented.

The games listed can be mounted from the web pages /index.ps3 or /games.ps3

If the auto-play option is checked in /setup.ps3, the game will be launched after mount it.

Additionally, the users can create custom html pages with /play.ps3 or /mount.ps3 to play their games (ISO or folders).
